packages/networks/endpoints.ts

```diff
@@ -2,9 +2,9 @@ import { NetworkEndpoints, Network } from './types'
 export const localhostEndpoints: NetworkEndpoints = {
   name: 'Localhost',
   id: Network.Localhost,
-  lcd: 'http://lcd.localhost',
-  tm: 'http://tm.localhost',
-  api: 'http://api.localhost'
+  lcd: 'https://lcd.localhost',
+  tm: 'https://tm.localhost',
+  api: 'https://api.localhost'
```

Comment:
1. For localhost, let's use http port though we still have https.
2. Use this network package in all chain & indexer examples to abstract away raw string address